doc/sidebar.md

```diff
@@ -0,0 +1,16 @@
+# Changing order of sidebar navigation items
+
+It is possible to change order of sidebar navigation items with configuration of app. Just add items names to property `sidebarItemsOrder` as list in that order which you want it to appear in sidebar: 
+
+```
+let ConfigDefaults = {
+	...
+	sidebarItemsOrder: ["Home", "Configuration", "Dashboard"]
```

Comment:
@titkov49 is `sidebarItemsOrder` takes the name of the `items` in the `Module`, am I right?

Reply:
@Pe5h4 yep, that's right

Reply:
Unfortunately, we cannot implement it this way - it assumes that application knows the final list of the sidebar items - which is non-systematic assumption.

It could be done e.g. by specifying a position "number" when adding item to the sidebar - then items are sorted by this number.

Reply:
I also was thinking about position number but if such would be implemented it would mean that in case if developer would need to change order of items in sidebar he might will need to go through each index.js of each module and change it's position which will make a lot of pain. Or if developer specify the same number for two modules than he can't control which of those two goes first and second which also isn't correct as I think.

Reply:
Okay, all that way along I was thinking about position number as integer but I understand now that position number can also be float which would solve issue that I described above. 
Anyway current ordering just throws all items that is not written in `sidebatItemsOrder` to the end of the list. In that way such ordering doesn't assume that it knows all items that appear or may appear in sidebar. It just sort them by list that is given from config. What I mean is that if we have three items in sidebar `["Home", "Table", "Dashboard"]` and in **sidebarItemsOrder** we specify only two from that list i.e. `sidebarItemsOrder=["Table", "Dashboard"]` then we will get sidebar with such ordering: `Table -> Dashboard -> Home`. 
If it isn't not what you meant then I can refactor that with position number.

Reply:
The position number is better for "composability" - e.g. in SeaCat WebUI, some modules are not loaded at all, so I don't know what is the final list of the items on the sidebar. But I can always define "approximate" order.
So if we say that you need to use number from 0-1000, then you know that small numbers are at the top and close to 1000 will be on the bottom. That will be fine for our goals.